lib/proofs.ts

```diff
@@ -156,3 +156,24 @@ export const sortProofsStatusAndTimes = (a: Proof, b: Proof) => {
   }
   return 0
 }
+
+export const downloadProof = (proof: Proof) => {
+  const { proof: binary, proof_id, block_number, team, machine_id } = proof
+
+  if (!binary) {
+    console.warn("Download failed - no proof binary found")
+    console.warn(`Block: ${block_number}, Proof ID: ${proof_id}`)
+    return
+  }
+
+  const teamName = team?.team_name ? team.team_name : machine_id.split("-")[0]
+  const suggestedFilename = `${block_number}_${teamName}_${proof_id}.bin`
```

Comment:
Fine for me at this early point but is the `.bin` used in this context? or should we just download the file without an extension?